azkaban-webserver/src/main/java/azkaban/webapp/servlet/ExecutorServlet.java

```diff
@@ -821,6 +829,15 @@ private void ajaxExecuteFlow(HttpServletRequest req,
     ret.put("execid", exflow.getExecutionId());
   }
 
+  /* Reset flow priority if submitting user is not a Azkaban admin */
+  private void fixFlowPriorityByPermission(ExecutionOptions options, User user) {
```

Comment:
ExecutorManager manager has no visibility into the permission level for the submit user. So we need to fix up set/unset properties in ExecutorServlet. Moreover we don't want all admin submitted flow to have high priority